Modified frontend to enhance UI #3 (under SWOC)

Samridhi012 opened the pull request:
Hey @Mahak008 !
I am delighted to have gained this opportunity to work with you and contribute to your amazing project. I am equally excited to see if you will like it. I have put my best efforts via contributing to #3 .
The changes made to enhance UI are as follows:
1. Fixed the navbar                                    
2. Touch-ups on 'Header' card via adding shadow to it and updates on its text
3. Sign-up form pop up updated
4. Cart enhanced
5. Menu enhanced
6. Enhancing Footer
7. Summing up, changed overall UI of project
Looking forward to a positive response. 
Thank you.

Mahak008 commented:
Hey @Samridhi012 , there are certain conflicts in this PR. Kindly resolve them and also share the screenshots of the changes you have made.

Samridhi012 commented:
Hi @Mahak008!
Conflicts has been resolved.

As indicated earlier, I have done following contributions:
1.Fixed the navbar
2.Touch-ups on 'Header' card via adding shadow to it and updates on its text
3.Sign-up form pop up updated
4.Cart enhanced
5.Menu enhanced
6.Enhancing Footer
7.Summing up, changed overall UI of project

And the screenshots are attached for the same in the following Google Drive link :
https://drive.google.com/drive/folders/1NsdZ8dIOqqOeqt0erqOMsSRfkwL1du1e?usp=drive_link

Thank you!

Mahak008 commented:
@Samridhi012 . It is still showing the conflicts.

Samridhi012 commented:
@Mahak008 please check if it is resolved now.
Conflicts in the file "Footer.css" was resolved earlier also but now even it is resolved.
Please check.

Samridhi012 commented:
@Mahak008 PR sent again after modification
